### ISSUE TITLE
Starting 

### ISSUE THREAD

**Issue by mohanbabi** [KenLagoni/OpenHD-LTE#1]
@KenLagoni  good morning
Today onwards, I need to test this code. I will make a components list according to your instrunctions. And I will post here. one your accepted those list I will go for Buy those.

Thanks you @KenLagoni sir

**Comment by KenLagoni** [comment]
I would wait a bit with this. I need to add more documentaiton and test the new h.264 transport code.

**Comment by mohanbabi** [comment]
Ok @KenLagoni  Sir.

**Comment by mohanbabi** [comment]
hi @KenLagoni  Sir. good morning. how are you? 
there is an "asterisk ". this use for VoIP applications and this is open source. link is  https://github.com/asterisk/asterisk

maybe using this can we make 4g telemetry effective.

**Comment by mohanbabi** [comment]
@KenLagoni  Sir how are you

**Comment by mohanbabi** [comment]
Hi @KenLagoni sir. how are you?

**Comment by KenLagoni** [comment]
> Hi @KenLagoni sir. how are you?

Thank you, all good here, just finishing the code for my h264 framing streaming protocol, first tests looks good.
Have you had any success with the 2G project?

**Comment by mohanbabi** [comment]
Yes, @KenLagoni  Sir 2G is a success, sir. Can I, test this 4g  now?

**Comment by KenLagoni** [comment]
> Yes, @KenLagoni Sir 2G is a success, sir. Can I, test this 4g now?
It is working, but I will admit my documentation is not 100% so it may be difficult:
- The Air part is ok, however I had to disable the recording on air due to performance issue when using my new h264 framing protocol.
- The ground pi (Recording) is also ok.
- The ground pi (OpenHD) is the one I still need to document, but my notes are in the "Modding OpenHD to groundstation.txt"

**Comment by mohanbabi** [comment]
@KenLagoni Sir, how can I reach you? like whatsapp. my number +91 9700701887

**Comment by KenLagoni** [comment]
> @KenLagoni Sir, how can I reach you? like whatsapp. my number +91 9700701887

Just added you :-)

**Comment by mohanbabi** [comment]
Thank you @KenLagoni  sir
